Modules/Core/Common/include/itkSymmetricEigenAnalysis.h

```diff
@@ -928,11 +925,11 @@ class ITK_TEMPLATE_EXPORT SymmetricEigenAnalysisFixedDimension
     {
     using ValueType = decltype(GetMatrixValueType(true));
     using EigenLibMatrixType =
-      Eigen::Matrix< ValueType, m_Dimension, m_Dimension, Eigen::RowMajor>;
-    EigenLibMatrixType inputMatrix(m_Dimension, m_Dimension);
-    for (unsigned int row = 0; row < m_Dimension; ++row)
+      Eigen::Matrix< ValueType, TMatrixDimension, TMatrixDimension, Eigen::RowMajor>;
+    EigenLibMatrixType inputMatrix(TMatrixDimension, TMatrixDimension);
```

Comment:
This constructor takes 'const unsigned int &'. The matrix is fixed so these parameters are redundant.